game.rb

```diff
@@ -0,0 +1,91 @@
+class Game
+
+board={A1: " ",B1: " ",C1: " ",A2: " ",B2: " ",C2: " ",A3: " ",B3: " ",C3: " "}
+
+def self.print_board(board)
```

Comment:
to nie powinna być metoda klasowa. Zobacz, że wszędzie jako parametr przekazujesz `board`.

To powinna być zmienna instancyjna tej klasy, a metody powinny ją zmieniać (bądź nie, zależnie która)